SU-HSDO/suhumsci
# Hsd8 251 custom login block

# READY FOR REVIEW

# Summary
- Customizable login block

# Needed By (Date)
- Sept 27

# Urgency
- Medium

# Steps to Test
1. checkout this branch
1. sync archaeology `blt drupal:sync --site=archaeology`
1. edit the layout on any basic page
1. click to add a block in any area of the page and choose the "HS Login Block" under the "H&S Blocks" category
1. configure as desired
1. Verify customizations & link are visible to logged out user.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
